ssr/choo.js

```diff
@@ -80,8 +76,8 @@ module.exports.render = function (app, route, cb) {
     var res = { state: state }
     res.body = app.toString(route, state)
     delete res.state._experimental_prefetch // State needs to be serializable.
-    if (app.state.title) res.title = app.state.title
-    if (app.state.language) res.language = app.state.language
+    if (state.title) res.title = state.title
```

Comment:
Could we maintain Choo 6 compatibility by doing something like `if (state.title || app.state.title)` here?

Reply:
Yes, that seems very reasonable 👍